src/js/components/stock-movement-wizard/inboundV2/sections/InboundV2AddItems.jsx

```diff
@@ -97,6 +107,41 @@ const InboundV2AddItems = ({
             />
 
             <div className="buttons-container">
+              <Button
+                onClick={() => {
+                  const fileInput = document.getElementById('csvInput');
+                  // Resetting fileInput.value to null ensures the onChange
+                  // event triggers even if the same file is selected again,
+                  // as browsers don't fire onChange for unchanged file inputs
+                  fileInput.value = null;
+                  fileInput?.click();
+                }}
+                StartIcon={<BiDownload className="icon" />}
+                defaultLabel="Import template"
+                label="react.default.button.importTemplate.label"
+                variant="primary-outline"
+              />
```

Comment:
Maybe it will be worth creating a separate component for uploading a file so that we can use the same one across the whole app?